packages/framework/src/Resources/views/Admin/Grid/Grid.html.twig

```diff
@@ -353,6 +353,10 @@
     {{ value|formatDateTime }}
 {% endblock %}
 
+{% block grid_value_cell_type_DateTimeImmutable %}
```

Comment:
Beware, commit 037b908280312bd62affc62d58abbe982db8c842 is fixuped to wrong commit and should be standalone commit anyway

Reply:
After discussion with @PetrHeinz we well squash ed8714c6a9cc884a7c6a1543e8c0b504b0c2a4ba e2a985401113feb2a0de00e1571c4c89fa446e68 037b908280312bd62affc62d58abbe982db8c842 442c2b570770660c8c607bc550ad9b8bcbe0a81b 02a6f051f22c5ef52808811829864233563b847b 04fb9706006de0bb2394e2d23aa31e6b767a7427

In one commit